### PR TITLE
style: reorder bottom navigation & change wording

### LINKED ISSUE #422: [UX] Bottom Page Navigation Improvements
## Description

there are several things that interfere with the bottom page navigation, including the location of the navigation items that are different from the previous version, and the use of mixed language.

126778941-3cee2218-fb7a-4cff-94a0-412b84f25d8e

`Pusat Data` used more often, but is in third place.

Capture

## Tasks

- [ ] Reorder navigation items according to the updated Figma file
- [ ] Rename menu items according to the updated Figma file.

### PR DESCRIPTION
**Pull request by ihsanrabb**
Closes #422 

## Current Tasks

- [x]  Reorder navigation items according to the updated [Figma file](https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485%3A2485)
- [x] Rename menu items according to the updated [Figma file.](https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485%3A2485)

### CODE REVIEW

**Review by resir014 — CHANGES_REQUESTED** [review]
The ordering is incorrect. Please take a look at the [updated Figma file](https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485%3A2485) for reference:

image

**Comment by ihsanrabb** [comment]
Base on @fncolon  description issue #422, Order Navigation Items Like Previous Bottom Page Navigation but it's different in  [updated Figma file](https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485%3A2485). Which order should we implement mas @resir014 ?

**Comment by resir014** [comment]
@ihsanrabb Let's follow the Figma. cc: @baraeb92

**Comment by resir014** [comment]
I've also updated the task list in the issue to avoid any confusion.

File: components/layout/navigation.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ export function Navigation() {
 
   const menuClasses = (isActive: boolean) => {
     return [
-      "inline-flex flex-col items-center justify-center h-12 px-4 rounded-md",
-      isActive ? "text-blue-600 font-semibold" : "text-gray-600",
+      "inline-flex flex-col items-center justify-center h-12 px-2 rounded-md",
+      isActive ? "text-blue-600 font-medium" : "text-gray-600",
```
Comment (resir014) [inline components/layout/navigation.tsx:18]:
```suggestion
      isActive ? "text-blue-600 font-semibold" : "text-gray-600",
```

The design on the Figma file uses a `font-size` of 600, which is `font-semibold`. Please keep it as it is.

image

**Review by resir014 — CHANGES_REQUESTED** [review]

File: components/layout/navigation.tsx [hunk 2]
```diff
@@ -27,13 +27,13 @@ export function Navigation() {
           className: "w-8 h-8",
           "aria-hidden": true,
         })}
-        <span className="text-xs">{item.name}</span>
+        <span className="text-xs text-center">{item.name}</span>
```
Comment (resir014) [inline components/layout/navigation.tsx:30]: @ihsanrabb Is this necessary? Didn't `items-center` above already sort the centering?

Comment (ihsanrabb) [reply components/layout/navigation.tsx:30]: I think it's better to add a `text-center`. `items-center` only center an icon on smaller screen @resir014 
bottom-nav

Comment (resir014) [reply components/layout/navigation.tsx:30]: Okay. Then I think we should move this up higher up the element tree, alongside `items-center justify-center` @ihsanrabb

**Review by resir014 — APPROVED** [review]

**Pull request merged**